packages/ckeditor5-package-generator/lib/utils/copy-files.js

```diff
@@ -86,6 +89,28 @@ function copyTemplate( templateFile, packagePath, data ) {
 	fs.writeFileSync( destinationPath, filledFile );
 }
 
+/**
+ * Creates the `.gitignore` file.
+ * See https://github.com/ckeditor/ckeditor5-package-generator/issues/50.
+ *
+ * @param {Options} options
+ */
+function createGitignore( options ) {
+	const gitignorePath = path.join( options.directoryPath, '.gitignore' );
+
+	const gitignoreComponents = [ GITIGNORE_ENTRIES ];
+
+	if ( options.programmingLanguage === 'ts' ) {
+		gitignoreComponents.push( GITIGNORE_ENTRIES_TS );
+	}
+
+	const gitignoreContent = gitignoreComponents
+		.map( components => [ ...components, '' ].join( '\n' ) )
+		.join( '\n' );
+
+	fs.writeFileSync( gitignorePath, gitignoreContent );
```

Comment:
```suggestion
	const ignoreEntries = [ ...GITIGNORE_ENTRIES ];

	if ( options.programmingLanguage === 'ts' ) {
		ignoreEntries.push( ...GITIGNORE_ENTRIES_TS );
	}

	fs.writeFileSync( gitignorePath, ignoreEntries.join( '\n' ) );
```

WDYT?

Reply:
This does not take into account empty lines between both sections and at the end of the file. We could of course add that empty line to the entry list, but it makes more sense for code to handle those additional gaps between sections.